app/jobs/regular/reopen_resolution.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Jobs
+  class ReopenResolution < ::Jobs::Base
+    def execute(args)
+      raise Discourse::InvalidParameters.new(:post_id) if args[:post_id].blank?
+
+      post = Post.find(args[:post_id])
+      Communitarian::Resolution.reopen_weekly_resolution!(post) if post
```

Comment:
we don't need this check since `Post.find` will raise exception if post will not be found
```suggestion
      Communitarian::Resolution.reopen_weekly_resolution!(post)
```